alacritty_terminal/src/term/mod.rs

```diff
@@ -1950,7 +1939,22 @@ impl ansi::Handler for Term {
                     cell.reset(&template);
                 }
             },
-            ansi::ClearMode::Saved => self.grid.clear_history(),
+            ansi::ClearMode::Below => {
+                for cell in &mut self.grid[self.cursor.point.line][self.cursor.point.col..] {
+                    cell.reset(&template);
+                }
+                if self.cursor.point.line < self.grid.num_lines() - 1 {
+                    self.grid
+                        .region_mut((self.cursor.point.line + 1)..)
+                        .each(|cell| cell.reset(&template));
+                }
+            },
+            ansi::ClearMode::All => {
+                self.grid.clear_viewport(&template);
+            },
+            ansi::ClearMode::Saved => {
+                self.grid.clear_history();
+            }
```

Comment:
These calls should be formatted to single line. Unfortunately rustfmt can't do this automatically so it needs to be done manually. Rustfmt can only break them when they are too long.